src/main/scala/fi/oph/koski/http/KoskiErrorCategory.scala

```diff
@@ -124,6 +124,8 @@ object KoskiErrorCategory {
         val alkamispäiväPuuttuu = subcategory("alkamispäiväPuuttuu", "Suoritukselle ei ole merkitty alkamispäivää")
         val valmiiksiMerkityltäPuuttuuOsasuorituksia = subcategory("valmiiksiMerkitylläEiOsasuorituksia", "Valmiiksi merkityllä päätason suorituksella ei voi olla tyhjä osasuorituslista tai opiskeluoikeuden pitää olla linkitetty")
         val tilaltaPuuttuuRahoitusmuoto = subcategory("tilaltaPuuttuuRahoitusmuoto", "Opiskeluoikeuden tilalta vaaditaan rahoitusmuoto")
+        val virheellinenOsaamisenHankkimistapa = subcategory("virheellinenOsaamisenHankkimistapa",
```

Comment:
Eri `ErrorCategory`llä on subcategoryja ja nyt tämä uusi virheviesti on `.validation.tila.virheellinenOsaamisenHankkimistapa`. `tila` subcategoryltä pitäisi löytyä vain tilaan liittyviä validaatoita. subcategory vaikuttaa virheviestin rakenteeseen minkä rajapinnan kutsuja saa. virheviestin voisi siirtää esimerkiksi `validation.rakenne` subcategoryn alle. myös tässä nimeäminen virheellinen->deprekoitu voisi ehkä auttaa integraatoiden tekijöitä ihmettelemään miksi aikaisemmin toimiva rakenne ei enää toimi.